FEWD/Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,87 @@
+var humanScore = 0;
+var computerScore = 0;
+
+function compChoice() {
+  var computerChoice = Math.random();
+  // console.log(computerChoice);
+  if (computerChoice <= 0.3) {
+    return computerChoice = "Rock";
+    // console.log(computerChoice);
```

Comment:
Consider removing `console.log`s once you're sure a certain part of the code works